client_plugin/drivers/shared/etcdops.go

```diff
@@ -269,8 +308,143 @@ func etcdService(cmd []string) {
 	}
 }
 
-func (d *VolumeDriver) createEtcdClient() *etcdClient.Client {
-	dclient := d.dockerd
+// checkLocalEtcd function check if local ETCD endpoint is successfully started or not
+// if yes, start the watcher for volume global refcount
+func (e *etcdInfo) checkLocalEtcd() error {
+	ticker := time.NewTicker(checkSleepDuration)
```

Comment:
These values are mainly set according to other projects. Will need to do more testing and adjust values to be more accurate.